website/templates/project/settings.mako

```diff
@@ -305,61 +309,69 @@
                     </div>
                     <div class="panel-body">
                         % if not node['institution']['name']:
-                            <div data-bind="visible: !loading()">
-                                <div data-bind="visible: error()">
-                                    <div class="help-block">
-                                        Could not load up available institutions. Please wait a few minutes and try again, or contact <a href="mailto:support@osf.io">support@osf.io</a> if the problem persists.
-                                    </div>
-                                </div>
-                                <div data-bind="visible: !error()">
-                                    <div data-bind="visible: availableInstitutions().length">
+                            %if 'admin' in user['permissions']:
```

Comment:
this is to prevent flashing of both while the request for the users list is being processed. Will probably be changed soon though, as user institutional information is being passed through mako variables.